pages/common/semgrep.md

```diff
@@ -0,0 +1,17 @@
+# semgrep
+
+> Perform *static source code analysis* (SAST) on your source code to detect security vulnerabilities.
+> Semgrep is a fast SAST tool aiming to minimize false positives.
+> More information: <https://semgrep.dev/>.
+
+- Run semgrep on a source directory, automatically detecting matching rules:
```

Comment:
```suggestion
- Run `semgrep` on a source directory, automatically detecting matching rules:
```

Reply:
There is no need to mention `semgrep` when the entire page about it.

Reply:
Can we use `it` instead over there?

Reply:
I'll go with this:

```suggestion
- Run on a source directory, automatically detecting matching rules:
```